thunder/benchmarks/targets.py

```diff
@@ -812,3 +814,29 @@ def test_resnet50(benchmark, executor: Callable, compute_type: ComputeType):
     fn = executor(b.fn())
 
     benchmark_for_compute_type(compute_type, benchmark, fn, args, kwargs)
+
+
+#
+# Thunder dynamo benchmarks
+#
+
+
+def test_dynamo_LlamaMLPBenchmark(benchmark):
```

Comment:
Is pytest-benchmark a test dependency to Thunder? I suggest putting this in the thunder/test directory because the main goal is testing `ThunderCompilerGraphBenchmarking`.

Reply:
The `ThunderCompilerGraphBenchmarking` requires pytest to create the bench fixture. I'll move it to `thunder/test`